docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md

```diff
@@ -53,6 +55,8 @@ There are some processors provided in `sumologic.yaml` that are intended to be u
 * **Memory limiter processor**. It is used to prevent out-of-memory situations on the collector. It should be always first on the processor's list. For more information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
 * **Batch processor**. It accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. See [Using batch processor to batch data](#using-batch-processor-to-batch-data) for more information.
 
+<!--There is another processor called ResourceDetection/system. Should that also be used?-->
```

Comment:
@angadrandhawa1 is this still needed?

Reply:
Yes we should call it our the ResourceDetection/System and explain what it does. @arunpatyal , @sumoanema can provide more details about the processor and how it should be used.

Reply:
The [resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/) can be used to detect resource information from the host,
in a format that conforms to the [OpenTelemetry resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions/), and append or override the resource value in telemetry data with this information. This processor is used to tag labels like :
* host.name
* host.id
* os.type

Reply:
@angadrandhawa1 @sumoanema thank you both!